mentorship iosTests/LoginTests.swift

```diff
@@ -110,6 +110,25 @@ class LoginTests: XCTestCase {
         
         //Test
         XCTAssertEqual(loginDisabledState, false)
+    }
+
+    func testLoginReset() {
+        let loginVM = LoginViewModel()
+
+        loginVM.loginData = LoginModel.LoginUploadData(username: "username", password: "password")
+
+        //MARK:- When reset is called, username and password should be reset
+
+        loginVM.resetLogin()
+
+        let currentLoginData = loginVM.loginData
+
+        //Test
+        XCTAssertEqual("", currentLoginData.username)
+        XCTAssertEqual("", currentLoginData.password)
+
+
+
```

Comment:
Since we're making changes in any case, can we remove these extra line breaks for neatness...

Reply:
done